lib/eco/eco.py

```diff
@@ -1265,8 +1336,121 @@ def toggle_menu(self, enabled=True):
 
         if enabled == False:
             self.ui.actionProfile.setEnabled(enabled)
+            self.ui.actionDebug.setEnabled(enabled)
             self.ui.actionStateGraph.setEnabled(enabled)
 
+    def debug_continue(self):
+        # pdb Command
```

Comment:
Since all these methods are called `debug_.*`  I think you can delete the comment `# pdb command`